zenodo/modules/thumbnails/cache.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Zenodo.
+# Copyright (C) 2019 CERN.
+#
+# Zenodo is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Zenodo is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Zenodo; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Implements a Redis cache."""
+
+from __future__ import absolute_import
+
+from flask_iiif.cache.redis import ImageRedisCache as ImageCache
+
+
+class ImageRedisCache(ImageCache):
```

Comment:
Since this is just a specialized Redis cache, I would keep the import name intact (i.e. `from flask_iiif.cache.redis import ImageRedisCache` and change this to:

```suggestion
class FilteredImageRedisCache(ImageRedisCache):
```